instagram/src/components/CommentSection/CommentSectionContainer.js

```diff
@@ -7,15 +7,31 @@ class CommentSection extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      comments: props.comments
+      comments: props.comments,
+      comment: ''
     };
   }
+commentTextHandler = e => {
+  this.setState=({comment: e.target.value});
+}
 
+commentSubmitHandler = e => {
```

Comment:
Nice job on the submit handler.  I would remove the alert before completion, but excellent step when working through initial functionality.